receiver/datadogreceiver/receiver_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"go.opentelemetry.io/collector/consumer"
```

Comment:
This is failing the linter, as we require the imports to be split in three blocks: stdlib, third-party, local packages. This here is a third-party among the stdlib. You can run `make lint` from the component's directory, and it should this:

```
datadogreceiver/receiver_test.go:11: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
	"go.opentelemetry.io/collector/consumer"
datadogreceiver/receiver_test.go:20: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
	"go.opentelemetry.io/collector/component/componenttest"
datadogreceiver/receiver_test.go:16: File is not `goimports`-ed with -local github.com/open-telemetry/opentelemetry-collector-contrib (goimports)
```

This is how it looks like:
```diff
diff --git a/receiver/datadogreceiver/receiver_test.go b/receiver/datadogreceiver/receiver_test.go
index 76ab51d4c3..573e271ba8 100644
--- a/receiver/datadogreceiver/receiver_test.go
+++ b/receiver/datadogreceiver/receiver_test.go
@@ -8,7 +8,6 @@ import (
        "context"
        "errors"
        "fmt"
-       "go.opentelemetry.io/collector/consumer"
        "io"
        "net/http"
        "strings"
@@ -18,6 +17,7 @@ import (
        "github.com/stretchr/testify/assert"
        "github.com/stretchr/testify/require"
        "go.opentelemetry.io/collector/component/componenttest"
+       "go.opentelemetry.io/collector/consumer"
        "go.opentelemetry.io/collector/consumer/consumertest"
        "go.opentelemetry.io/collector/pdata/pcommon"
        "go.opentelemetry.io/collector/pdata/pmetric"
```